Functions/xASL_quant_Basil.m

```diff
@@ -46,28 +46,33 @@
     end
     fprintf('%s\n', 'Note that any file not found warnings can be ignored, this pertains to the use of symbolic links by BASIL');
     
+    % Remove residual BASIL-related files
     xASL_delete(pathBasilOptions);
     xASL_delete(pathBasilInput);
-    xASL_adm_DeleteFileList(x.dir.SESSIONDIR, '(?i)^.*basil.*$', 1, [0 Inf]); % remove residual BASIL-related files
+    xASL_adm_DeleteFileList(x.dir.SESSIONDIR, '(?i)^.*basil.*$', 1, [0 Inf]);
     
     %% 3. Write the PWI as Nifti file for Basil to read as input
-    % FIXME would be good to have a brain mask at this point
-    % PM: if this would be a brainmask as well, we can skip creating a
-    % dummy input image here
+    % FIXME would be good to have a brain mask at this point -> PM: if this would be a brainmask as well, we can skip creating a dummy input image here
```

Comment:
Is that comment still relevant? I personally don't mind it, but I wasn't sure if it's still up-to-date.

Reply:
BASIL requires a brain mask, and we create specific mask(s) in ExploreASL, so this comment is there to remind us to see if this can be synchronised better, for better comparability between the results of our quantification, and BASILs quantification.